### PR TITLE
Glitch fixes

### PR DESCRIPTION
**Pull request by karimbeyrouti**
Fixed rendering issues in Internet Explorer 10
Updated rotateY SASS notation to explicitly use degrees

### CODE REVIEW

File: sass/heidelberg/_heidelberg-plugin.sass [hunk 1]
```diff
@@ -120,3 +102,18 @@ $_right: odd
 // Needs to be on own line to work in IE8
 .no-csstransforms3d .Heidelberg-Page.with-Spread.is-active + .Heidelberg-Page.with-Spread.is-active .Heidelberg-Spread
   left: -100%
+
+// Internet 10 only selector - checks Heidelberg HTMLElement attribute: data-useragent
+.Heidelberg-Book[data-useragent*='MSIE 10.0']
```
Comment (callumacrae) [inline sass/heidelberg/_heidelberg-plugin.sass]: What does this do and why shouldn't it be applied to other browsers?

Comment (karimbeyrouti) [inline sass/heidelberg/_heidelberg-plugin.sass]: Because this fixes a rendering issue only in IE 10.0 - All other version of IE are fine.

Comment (callumacrae) [inline sass/heidelberg/_heidelberg-plugin.sass]: What happens if you apply the fix to the other browsers? Does it break or does everything stay the same? I'd prefer it if there weren't any browser-specific code.

Comment (karimbeyrouti) [inline sass/heidelberg/_heidelberg-plugin.sass]: I would would also prefer if there were no browser-specific code, unfortunately the fix did break all other browsers.

Comment (callumacrae) [inline sass/heidelberg/_heidelberg-plugin.sass]: Fair enough. @barneyfoxuk: Are we still using this repo, or are we going to use the version on the lostmyname org?

Comment (barneyfoxuk) [inline sass/heidelberg/_heidelberg-plugin.sass]: Probably best to use LMN one now

**Comment by karimbeyrouti** [comment]
In that case I will also send PR to LMN repo....

**Pull request closed without merging**